From 5ca05c3c20e74046290ce2ef11c2ef95ad08e37b Mon Sep 17 00:00:00 2001
From: Geetha sowjanya <gakula@marvell.com>
Date: Wed, 8 May 2019 16:14:20 +0530
Subject: [PATCH 025/138] octeontx2-pf: Set minimum MTU size to 64 bytes

commit fdfca30496bb0152daaf7d1f4f83ce78d6344b66 from
git@git.assembla.com:cavium/WindRiver.linux.git

Some old systems expect mini packet size to be 64B.
Hence, changing min MTU size to 64B to support them.

Change-Id: I8234f317ce7e71672251f70dd6564f8777382672
Signed-off-by: Geetha sowjanya <gakula@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/8743
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/nic/otx2_txrx.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/nic/otx2_txrx.h b/drivers/net/ethernet/marvell/octeontx2/nic/otx2_txrx.h
index e2dc87608459..411250a9456c 100644
--- a/drivers/net/ethernet/marvell/octeontx2/nic/otx2_txrx.h
+++ b/drivers/net/ethernet/marvell/octeontx2/nic/otx2_txrx.h
@@ -28,7 +28,7 @@
 #define OTX2_HEAD_ROOM		OTX2_ALIGN
 
 #define	OTX2_ETH_HLEN		(VLAN_ETH_HLEN + VLAN_HLEN)
-#define OTX2_MIN_MTU		ETH_MIN_MTU
+#define OTX2_MIN_MTU		64
 #define OTX2_MAX_MTU		(9212 - OTX2_ETH_HLEN)
 
 #define OTX2_MAX_GSO_SEGS	255
-- 
2.17.1

